### PR TITLE
Preliminary OpenShift Virtualization support

### PR DESCRIPTION
**Pull request by gqlo**
Preliminary OpenShift Virtualization support.  Workloads are not supported yet.

### CODE REVIEW

File: clusterbuster [hunk 1]
```diff
@@ -710,6 +717,12 @@ function process_option() {
 		*) affinity=0         ;;
 	    esac
 	    ;;
+        # vm specs
+        vmcores)                    vm_cores=$optvalue                          ;;
+        vmsockets)                  vm_sockets=$optvalue                        ;;
+        vmthreads)                  vm_threads=$optvalue                        ;;
+        vmgraceperiod)              vm_grace_period=$optvalue                   ;;
+        vmimage)                    vm_image=$optvalue                          ;;
```
Comment (RobertKrawitz) [inline clusterbuster:725]: These options need to be documented in _help_extended.  Make certain that when you add them there that you use only spaces for indentation, not tab characters.

**Review by RobertKrawitz — CHANGES_REQUESTED** [review]
A few comments, but close.

File: lib/clusterbuster/workloads/server.workload [hunk 2]
```diff
@@ -190,7 +190,7 @@ function server_supports_reporting() {
 function server_report_options() {
     cat <<EOF
 "msg_size": $___msg_size,
-"interface": "${___server_interface}"
+"interface": "${___server_interface}",
```
[the remark this answers is not in the document]
Comment (gqlo) [reply lib/clusterbuster/workloads/server.workload]: Merged

File: examples/clusterbuster/memory [hunk 3]
```diff
@@ -9,4 +9,4 @@ no-report-object-creation
 replicas=8
 processes=3
 namespaces=1
-memorysize=1Gi
+memorysize=512Mi
```
[the remark this answers is not in the document]
Comment (gqlo) [reply examples/clusterbuster/memory]: Merged against the main

File: clusterbuster (hunk 1 above)
Comment (gqlo) [reply clusterbuster:738]: added the help messages under Virtualization Options.

File: clusterbuster [hunk 4]
```diff
@@ -436,6 +443,19 @@ $(print_workloads_supporting_reporting '                        - ')
                         Allow use of direct I/O for virtiofsd (default $virtiofsd_direct).
        --virtiofsd-threadpoolsize=n
                         Use the specified thread pool size for virtiofsd (default 1).
+       --vmthreads=<value>
+                        Specify the number of threads on each core (default $vm_threads).
```
Comment (RobertKrawitz) [inline clusterbuster:447]: Please put this in a separate section.  Please also rename "Virtualization Options" to "Kata Virtualization Tuning".

**Review by RobertKrawitz — CHANGES_REQUESTED** [review]

File: clusterbuster (hunk 4 above)
Comment (gqlo) [reply clusterbuster:449]: Just did as suggested.

**Review by RobertKrawitz — APPROVED** [review]

**Pull request merged**